feat: add unit tests for drawValues in HorizontalBarChartRenderer

amaekh opened the pull request:
fixes #16

amaekh committed:
feat: add unit tests for drawValues in HorizontalBarChartRenderer

HaraldHR reviewed (changes requested):
I think it looks good, the tests work. I think however we should merge it into the unittests branch first since we haven't fully decided on exactly what we want to have in our master branch. I also noticed that I had to add mockito as a dependency since it wasn't in the build.gradle file in your branch (I assume that's because you just didn't add that change to the commit), but that should be fine because in the unittests branch I've already added it, so they should work without additional setup there.

amaekh commented:
> I think it looks good, the tests work. I think however we should merge it into the unittests branch first since we haven't fully decided on exactly what we want to have in our master branch. I also noticed that I had to add mockito as a dependency since it wasn't in the build.gradle file in your branch (I assume that's because you just didn't add that change to the commit), but that should be fine because in the unittests branch I've already added it, so they should work without additional setup there.

Oh right, I missed to stage that as well! Now I have changed it to the unittest branch, thank you!

HaraldHR reviewed (approved):
Good job!
